robottelo/test.py

```diff
@@ -97,6 +98,23 @@ class TestCase(unittest2.TestCase):
     cleanup_queue = defaultdict(deque)
     deleted_entities = defaultdict(set)
 
+    @pytest.fixture(autouse=True)
+    def set_worker_id_logger(self, worker_id):
```

Comment:
Since this is something internal to robottelo what about using `_` prefix?

That will prevent the being override on children test cases, for example a test that deals with some Satellite workers can easily override the worker_id.